### PR TITLE
fix(ground_segmentation launch): fix ground segmentation concatenate node and timeout

### PR DESCRIPTION
**Pull request by badai-nguyen**
## Description

-  To add timeout in cases multi-additional lidars used for ground_segmentation

## Releated Link

- Related PR: https://github.com/tier4/aip_launcher/pull/215/files

- [TIER IV INTERNAL LINK](https://tier4.atlassian.net/browse/RT1-5185?focusedCommentId=165666)

## Tests performed
- Tested on X2 bench hardware: `/perception/obstacle_segmentation/concatenate_data` output (`/perception/obstacle_segmentation/single_frame/pointcloud_raw`) rate:  7.2hz -> 10hz

| Before | After |
| --- | --- |
|Screenshot from 2024-05-02 11-34-36 | Screenshot from 2024-05-02 09-03-38|

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: launch/tier4_perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py [hunk 1]
```diff
@@ -445,6 +445,7 @@ def get_additional_lidars_concatenated_component(input_topics, output_topic):
                     "input_topics": input_topics,
                     "output_frame": LaunchConfiguration("base_frame"),
                     "input_twist_topic_type": "odom",
```
Comment (YoshiRi) [inline launch/tier4_perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py]:
```suggestion
                    "approximate_sync": True,
```

File: launch/tier4_perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py [hunk 2]
```diff
@@ -434,7 +434,7 @@ def create_single_frame_outlier_filter_components(input_topic, output_topic, con
     def get_additional_lidars_concatenated_component(input_topics, output_topic):
         return ComposableNode(
             package="pointcloud_preprocessor",
-            plugin="pointcloud_preprocessor::PointCloudConcatenateDataSynchronizerComponent",
+            plugin="pointcloud_preprocessor::PointCloudConcatenationComponent",
             name="concatenate_data",
             remappings=[
                 ("~/input/odom", "/localization/kinematic_state"),
```
Comment (YoshiRi) [inline launch/tier4_perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py]:
```suggestion
```

**Review by YoshiRi — CHANGES_REQUESTED** [review]
Please fix requested part :bow:

File: launch/tier4_perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py (hunk 2 above)
Comment (badai-nguyen) [reply launch/tier4_perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py]: fixed: https://github.com/tier4/autoware.universe/pull/1283/commits/d3e728d623875b13324e8f656612a1f15529bf3a

File: launch/tier4_perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py (hunk 1 above)
Comment (badai-nguyen) [reply launch/tier4_perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py]: I fixed at https://github.com/tier4/autoware.universe/pull/1283/commits/d3e728d623875b13324e8f656612a1f15529bf3a

**Review by YoshiRi — APPROVED** [review]
LGTM

**Review by yukkysaito — APPROVED** [review]

**Pull request merged**